content/2022/06-30-improvements-in-web-app-integrations-documentation.md

```diff
@@ -0,0 +1,29 @@
+---
+applied_at: "2021-06-30"
+applies_to: 
+- other
+is_impactful: true
+is_new_feature: false
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# Improvements in web app integrations documentation
```

Comment:
```suggestion
# Improvements in Web App Integrations documentation
```
I would capitalize the title to match the doc title (https://developer.box.com/guides/applications/web-app-integrations/configure/) although I'm not really sure if we need to capitalize it in general :D